src/effects/chapters.js

```diff
@@ -31,7 +31,9 @@ const parseChapters = duration => (result, chapter, index, chapters) => {
       start: toPlayerTime(chapter.start),
       end: toPlayerTime(end.start),
       title: get(chapter, 'title'),
-      image: get(chapter, 'image')
+      image: get(chapter, 'image'),
+      href: get(chapter, 'href') ? get(chapter, 'href') : null,
+      link_title: get(chapter, 'href') ? new URL(get(chapter, 'href')).hostname.replace(/^(www\.)/, '') : null
```

Comment:
it's just cosmetic but I try to use lower camel case for object attributes :)

Reply:
Of course. Changing that. I made it an extra property here instead of calculating it later to maybe also expose it into the chapter json/object on config. Do you think that's prudent?